tests/scripts/task_python_integration.sh

```diff
@@ -25,22 +25,17 @@ export LD_LIBRARY_PATH="build:${LD_LIBRARY_PATH:-}"
 rm -rf python/tvm/*.pyc python/tvm/*/*.pyc python/tvm/*/*/*.pyc
 
 # Test TVM
-make cython
 make cython3
 
 # Test extern package
 cd apps/extension
 rm -rf lib
 make
 cd ../..
-python -m nose -v apps/extension/tests
```

Comment:
should change to `python3` instead of deleting, right?

Reply:
Fixed.